src/main/res/values/strings.xml

```diff
@@ -1089,4 +1089,20 @@
 	<string name="pref_behaviour_hide_read_posts_key" translatable="false">pref_behaviour_hide_read_posts</string>
 	<string name="pref_behaviour_hide_read_posts_title">Hide read posts</string>
 
+	<!-- 2018-10-06 -->
+	<string name="mainmenu_shortcut_all">All subreddits</string>
+	<string name="mainmenu_shortcut_custom">Custom Location</string>
+	<string name="mainmenu_shortcut_frontpage">Frontpage</string>
+	<string name="mainmenu_shortcut_random">Random</string>
+	<string name="mainmenu_shortcut_popular">Popular</string>
```

Comment:
For the five strings above, I think we should just use the existing ones (`mainmenu_frontpage`, `mainmenu_all`, etc)